subsystems/drivebase/SwerveDrive.java

```diff
@@ -479,4 +511,19 @@ public boolean getAsBoolean() {
         };
         return new MustangPPSwerveControllerCommand(path, this::getPose, this::getChassisSpeeds, this::driveRobotRelative, config, alliance, reqSubsystems);
     }
+
+    @Override
+    public void debugSubsystem() {
+        if (kConfig.kDriveMotorType.equals(Motor_Type.NEO)) {
```

Comment:
I don't understand the point of this. This will record the output current from the motor controller to the motor. Which... ok. But it's irrelevant to solving the problem of drawing too much current from the battery on the input side of the motor controller